src/Inline/Element/AbstractInlineContainer.php

```diff
@@ -44,9 +39,9 @@ public function getChildren()
      */
     public function setChildren($contents)
     {
-        $this->children = $contents;
-        foreach ($contents as $content) {
-            $content->setParent($this);
+        $this->detachChildren();
+        foreach ($contents as $children) {
+            $this->appendChild($children);
```

Comment:
Well, this method used only because of `Image` and `Link` requires label nodes to be passed to constructor. So yes, I can just drop this method.